### PR TITLE
10.0 pos ms and pos pa compatibility

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_product_available/static/src/js/pos.js [hunk 1]
```diff
@@ -46,6 +46,23 @@ odoo.define('pos_product_available.PosModel', function(require){
...
+
+        update_product_qty_from_order_lines: function(order) {
+            var self = this;
+            order.orderlines.each(function(line){
+                var product = line.get_product();
+                product.qty_available -= line.get_quantity();
+                self.refresh_qty_available(product);
+            });
+            // compatibility with pos_multi_session
+            order.trigger('new_updates_to_send');
```
Comment (yelizariev) [inline pos_product_available/static/src/js/pos.js:62]: It not a part of the [instruction](https://github.com/it-projects-llc/pos-addons/tree/10.0/pos_multi_session#modules-compatibility). Could you update instruction please?

File: pos_product_available/static/src/js/pos.js [hunk 2]
```diff
@@ -46,6 +46,23 @@ odoo.define('pos_product_available.PosModel', function(require){
             });
         },
 
+        set_product_qty_available: function(product, qty) {
+            product.qty_available = qty;
+            this.refresh_qty_available(product);
+        },
```
Comment (yelizariev) [inline pos_product_available/static/src/js/pos.js:52]: Is it part of compatibility  or something else?

Comment (KolushovAlexandr) [reply pos_product_available/static/src/js/pos.js:52]: @yelizariev I've refactored the code a bit, there were too many repeated stuff

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
We have got a synchronization of available product quantity.

File: pos_multi_session/README.rst [hunk 3]
```diff
@@ -120,6 +120,22 @@ The code below is a real example from module `pos_order_note <https://www.odoo.c
...
+.. code-block:: js
+
+    update_product_qty_from_order_lines: function(order) {
+        var self = this;
+        order.orderlines.each(function(line){
+            var product = line.get_product();
+            product.qty_available -= line.get_quantity();
+            self.refresh_qty_available(product);
+        });
+        // compatibility with pos_multi_session
```
Comment (yelizariev) [inline pos_multi_session/README.rst]: Explain what it does or why it's needed

File: pos_multi_session/README.rst [hunk 4]
```diff
@@ -120,6 +120,22 @@ The code below is a real example from module `pos_order_note <https://www.odoo.c
         }
     });
 
+Also it's possible to trigger multi_session events from other modules, for example code from `pos_product_available <https://www.odoo.com/apps/modules/10.0/pos_product_available/>`__:
```
Comment (yelizariev) [inline pos_multi_session/README.rst]: Why?

**Pull request merged**